### PR TITLE
Add missing browser.safebrowsing.provider.mozilla.lists pref

### PR DESCRIPTION
**Pull request by ericawright**
@fmarier 

fixes: #109

### CODE REVIEW

**Comment by fmarier** [comment]
Looks good to me.

I just thought of another pref that would be good to include: `urlclassifier.disallow_completions`

The change would be the same as what you just did with `browser.safebrowsing.provider.mozilla.lists`: take the default value and add the fastblock lists we need

e.g. for L1: `urclassifier.disallow_completion += ",fastblock1-track-digest256,fastblock1-trackwhite-digest256"`

**Comment by fmarier** [comment]
FYI: I'm also trying to include all of the FastBlock lists directly on central: https://bugzilla.mozilla.org/show_bug.cgi?id=1489590

This patch may not get uplifted in time for the study, so we should still set these prefs in the extension.

**Comment by ericawright** [comment]
@fmarier thank you, updated

File: src/variations.js [hunk 1]
```diff
@@ -90,39 +90,6 @@ window.VARIATIONS = {
     },
   },
 
-  "FB2L2": {
```
Comment (fmarier) [inline src/variations.js]: I think you removed the wrong one. We do need FB2L2.

Comment (ericawright) [reply src/variations.js]: omg I did, sorry

**Comment by fmarier** [comment]
I'd suggest also adding this to all variations:

    browser.safebrowsing.provider.mozilla.nextupdatetime = 1

to help download the lists faster.

**Comment by fmarier** [comment]
Other than the FB2L2 which I mentioned in https://github.com/mozilla/FastBlockShield/pull/115#discussion_r216104887 and that @ericawright has already fixed, I manually tested all of the variations and they all successfully download the required lists once an update is triggered in `about:url-classifier`.

**Pull request merged**